src/core/constants.ts

```diff
@@ -1,12 +1,21 @@
+export interface Nis1NetworkType {
+    name: string;
+    identifier: number;
+}
+
+export interface SymbolNetworkType extends Nis1NetworkType {
```

Comment:
The Network class is an abstract class takes only name and identifier. There's no need to split the types at all imo

Reply:
sure, the reason I split it because `generationHash` is only used for Symbol.